syntax = "proto3";

package hello_world;

service HelloWorld {
  rpc say_hello_world(HelloWorldRequest)
    returns (HelloWorldRespond);
}

message HelloWorldRequest {
  string user_name = 1;
}

message HelloWorldRespond {
  string text = 1;
}
